server/app/services/applicant/question/AddressQuestion.java

```diff
@@ -244,6 +261,10 @@ public Path getWellKnownIdPath() {
     return applicantQuestion.getContextualizedPath().join(Scalar.WELL_KNOWN_ID);
   }
 
+  public Path getServiceAreaPath() {
```

Comment:
**suggestion (non-blocking, if-minor):** memoize

Reply:
memoized

Reply:
I'm not seeing that for this method?

Reply:
updated